test/test_watch_schedule_builder/test_per_week_limit.py

```diff
@@ -0,0 +1,29 @@
+from datetime import date
+from ketchup import watch_schedule_builder as wsb
+
+
+def test_per_week_limit_none():
```

Comment:
these are looking good! we can talk through them and make a separate ticket to expand them as well :)

Reply:
Still trying to build confidence with tests, but I felt like these were simple enough to test the process.

Reply:
lets pull your new tests out of this PR so we can get tests running :)

Reply:
Will require your assistance in that, because I am unsure on what you wish for me.

Reply:
You can remove these changes from this PR in multiple ways. But essentially, you should create a new branch and move these tests to that branch and submit it separately. This way we can get the code we want in sooner without waiting for this other feature to finish. 

If, say, the test file is on one commit and the per week changes were on another, you could "cherry pick" the commit into another. If it's in the same commit, you might have to create a new branch then checkout the test file from this branch... something like "git checkout <this branch name> <the test file or test dir>" which will move the test file to the new branch. Then you can PR that branch and still get your tests in 😌